src/orquestra/integrations/cirq/simulator/qsim_simulator.py

```diff
@@ -55,7 +55,7 @@ def __init__(
         qubit_order=cirq.ops.QubitOrder.DEFAULT,
         seed: cirq.RANDOM_STATE_OR_SEED_LIKE = None,
         circuit_memoization_size: int = 0,
-        qsim_options: Optional[Union[Dict, "qsimcirq.QSimOptions"]] = None,
+        qsim_options: Optional["qsimcirq.QSimOptions"] = None,
```

Comment:
Actually, why it's string and not just `qsimcirq.QSimOptions` ?
Or even just `QsimOptions` with an import at the top?
```
from qsimcirq import QsimOptions
```

Reply:
Mypy was throwing an error and @dexter2206 showed this as a method to overcome this issue.